modules/apps/frontend-js/frontend-js-dropdown-support-web/src/main/resources/META-INF/resources/DropdownProvider.js

```diff
@@ -97,6 +97,21 @@ class DropdownProvider {
 		trigger.parentElement.classList.add(CssClass.SHOW);
 		trigger.setAttribute('aria-expanded', true);
 
+		const clickOutsideHandler = (event) => {
+			if (
+				!menu.contains(event.target) &&
+				!trigger.container(event.target)
```

Comment:
is `Node.container()` a method that I'm unaware of or is this a typo?

Reply:
typo 🤦 🤦 🤦 🤦

Reply:
Thanks for point up this!

Reply:
Relief. I was about to go study the new DOM spec 🤣

Reply:
Hahahahaha! I didn't notice it either!